### ISSUE TITLE
Embrase!

### ISSUE THREAD

**Issue by paazmaya** [stephenplusplus/sindredash#1]
Good stuff! :santa:

**Comment by rex** [comment]
Indeed. This is the _perfect_ repo for lazy programmers like me. You're a badass, @sindresorhus, but like @stephenplusplus I don't have all day to define all 27,946 of your repos as dependencies. So now I can have my cake and eat it too :rocket: :+1:
